catboost/R-package/configure.ac

```
@@ -0,0 +1,91 @@
+# autoconf/configure script for catboost
+
+##### INIT ####################################################################
+
+AC_INIT(catboost, 0.12, nikitxskv@yandex-team.ru)
```

Autoconf allows dynamically generate version number (https://stackoverflow.com/questions/8559456/read-a-version-number-from-a-file-in-configure-ac), probably it is better to read it from DESCRIPTION file

---

yes and yes.
```suggestion
AC_INIT(catboost,
        m4_esyscmd([cat DESCRIPTION | grep Version: | sed -e 's/^Version: \([0-9]*[.][0-9]*\).*$/\1/' | tr -d '\n']),
        [https://github.com/catboost/catboost/issues])
```

This will do the trick